plans/artifactory-pro/hooks/run

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+echo "Starting Artifactory Pro"
+
+export JAVA_HOME=$(hab pkg path core/server-jre)
+export ARTIFACTORY_HOME={{pkg.svc_var_path}}
+
+{{pkg.svc_var_path}}/bin/artifactory.sh 2>&1
```

Comment:
Things begin to get pretty unreadable at some point. I would love to hear a comparison on using `pkg_svc_run` vs a `run` hook. @fnichol ?